### DISCUSSION TITLE
Update site with Tycho?

### DISCUSSION THREAD

**Discussion by rfolo9li** [eclipse-tycho/tycho#1108] [Q&A]
I have a traditional plugin project which consists of multiple plugin/bundles. These plugins are bundled by various feature bundles and these feature bundles in turn are included in a update site project. In other terms, the structure is: An update site includes multiple feature bundles which include multiple plugins each.

How do I setup a update site with Tycho for this?

There is the tycho-packaging plugin. Is that the plugin I need to use? If so: Which goal do I need? The plugin has the goals update-site-packaging and update-site and I don't understand the difference.

**Comment by cdietrich** [comment]
You can check 
https://github.com/itemis/xtext-reference-projects/tree/master/greetings-tycho/2.28.0/org.xtext.example.mydsl.repository

creating a category.xml and using `<packaging>eclipse-repository</packaging>` should be sufficient
